pkgs/by-name/sn/snowflake-cli/package.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "snowflake-cli";
+  version = "3.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "snowflakedb";
+    repo = "snowflake-cli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-QBYdzKKkakgwCD12bN7CerjfvMnxvt2cJ0NqTIjHkWw=";
+  };
+
+  build-system =
+    (with python3Packages; [
+      hatch-vcs
+      hatchling
+      pip
+    ])
+    ++ [ installShellFiles ];
```

Comment:
```suggestion
  build-system = with python3Packages; [
      hatch-vcs
      hatchling
      pip
    ];

  nativeBuildInputs = [ installShellFiles ];
```

Apply format.

Reply:
Thanks! Applied format and moved installshellfiles to nativebuildinputs.